lib/storage/shared.js

```diff
@@ -487,6 +490,54 @@ async function issueNoteChallenge(client, requestParams, body = {}) {
   return request.json()
 }
 
+async function createGroupRequest(
+  crypto,
+  data,
+  encryptionKeyPair,
+  recipientEncryptionKey
+) {
+  const accessKey = await crypto.randomKey()
```

Comment:
This should be a keypair and not just a single key

Reply:
https://github.com/tozny/e3db-clients-go/blob/e0e19af0ad0c22931e2aca4b3ea6ea7fb09736d6/storageClient%2FstorageClient_test.go#L280 is a good reference